python/build-requirements.txt

```diff
@@ -17,6 +17,7 @@ pandas==1.2.1
 pytest==6.2.1
 pytest-cov==2.11.0
 pytest-html==3.1.1
+scikit-learn==0.21.3
```

Comment:
If we are using a third party library (which we probably shouldn't for this case), let's make sure it is a pre-approved version.